lib/pandadoc/documents/documents.ex

```diff
@@ -17,6 +17,10 @@ defmodule Pandadoc.Documents do
 
   @type valid_document_statuses() :: 2 | 10 | 11 | 12
 
+  @type change_signer_attrs :: %{
+          id: String.t()
+        }
```

Comment:
@seanhuggins1 who does this `id` belong to? is this the `user_id` for the new signer?

Reply:
yes, in their api it's just called `id` in the body parameter